backend/internal/usecase/transaction.go

```diff
@@ -228,19 +229,21 @@ func (uc *TransactionUsecase) QueryTransactionDetailsByTxHash(ctx context.Contex
 			}
 		} else if tse.TxStatus.TxStatusVal == data.TxStatusValUnshieldedinclusion {
 			txStatus = UnshieldedInclusion
-			effectiveInclusionTime = tse.DecryptedTxCreatedAtUnix
```

Comment:
Could we add the inclusion information here as well since this is also considered an inclusion, i.e. the effective inclusion time, the inclusion delay, as well as the block number, and remove the inclusion slot as this is not relevant to the unshielded inclusion?

Reply:
Could we additionally change the estimated inclusion time to a Unix timestamp as this is supposed to help the user see when exactly their transaction will be included, which is then replaced with the effective inclusion time?